product/infra/src/docker.ts

```diff
@@ -7,6 +7,20 @@ export const deployContainer = (config: IDockerSettings, dbConnectionString: str
     // Location to deploy Cloud Run services
     const location = gcp.config.zone ?? ""
 
+    const githubClientId = "11c2d39160e19c5d2ab0"
```

Comment:
maybe have it as secrets? or not needed?

Reply:
same as above